utils/hct/hctbuild.cmd

```diff
@@ -188,16 +188,6 @@ if "%1"=="-dxc-cmake-system-version" (
 if "%1"=="-show-cmake-log" (
   set SHOW_CMAKE_LOG=1
   shift /1 & goto :parse_args
-)  
-if "%1"=="-disable-lit" (
-  echo Disable LIT testing
-  set ENABLE_LIT=OFF
```

Comment:
It seems like there should be more cleanup than this...

Reply:
Updated.